### PR TITLE
feat: Merge 0.46.x-celestia on QGB feature branch

### PR DESCRIPTION
**Pull request by SweeXordious**

### CODE REVIEW

**Review by evan-forbes — COMMENTED** [review]
had one blocking question

would updating the sdk help at all?

File: Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ SIMAPP = ./simapp
 MOCKS_DIR = $(CURDIR)/tests/mocks
 HTTPS_GIT := https://github.com/cosmos/cosmos-sdk.git
 DOCKER := $(shell which docker)
-DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.0.0-rc8
+DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf:1.7.0
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply Makefile:15]: I tried to keep the changes similar to upstream and not skip a version and end up breaking compatibility without knowing.
I can bump the version if needed tho

**Comment by SweeXordious** [comment]
> would updating the sdk help at all

I am not quite sure what you mean. Do you mean to have the CI passing? The CI won't pass as it is ran against the target branch which doesn't have the latest `buf` version.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (evan-forbes) [reply Makefile:15]: that's fine, I didn't know upstream also has the their docker using v1.7.0 while their CI uses v1.8.0

**Comment by evan-forbes** [comment]
> Do you mean to have the CI passing?

yeah, like, if we upgraded to v0.46.1, would it fix this. That version uses v1.4.0, so I'm unsure. I'm fine with using this.

**Review by evan-forbes — APPROVED** [review]

**Comment by SweeXordious** [comment]
In that case we check if they bumped the version in that branch. If not, we can do it ourselves. Shouldn't be a problem, I guess

**Pull request merged**